dashboard/test/controllers/certificate_images_controller_test.rb

```diff
@@ -32,6 +32,9 @@ class CertificateImagesControllerTest < ActionController::TestCase
   end
 
   test 'can show course1 course name' do
+    course1 = Unit.find_or_create_by!(name: 'course1', is_course: true)
```

Comment:
course1 exists in our fixtures, but the course version does not. I'm doing `find_or_create_by!` here so that this test will keep passing when we eventually get rid of the fixtures.